app/src/main/java/com/iyanuoluwa/imovie/NowPlaying.kt

```diff
@@ -12,9 +12,9 @@ import androidx.fragment.app.Fragment
 import androidx.recyclerview.widget.GridLayoutManager
 import androidx.recyclerview.widget.LinearLayoutManager
 import androidx.recyclerview.widget.RecyclerView
-import com.iyanuoluwa.imovie.adapter.MovieAdapter
-import com.iyanuoluwa.imovie.api.MovieJson
-import com.iyanuoluwa.imovie.data.ApiNowPlaying
+import com.iyanuoluwa.imovie.ui.adapter.MovieAdapter
+import com.iyanuoluwa.imovie.data.model.buffer.api.MovieJson
```

Comment:
I see this `buffer` and I'd like to point out that it's unnecessary. They used it in the example you learnt from because that's the name of their site. Also, the `api` package isn't needed here as well, and this is a small app, so you can just put all your models directly under the `model` package

```suggestion
import com.iyanuoluwa.imovie.data.model.MovieJson
```